config/config.go

```diff
@@ -548,6 +550,7 @@ func DefaultMempoolConfig() *MempoolConfig {
 		// ABCI Recheck
 		Size:      5000,
 		CacheSize: 10000,
+		MaxBytes:  1000000000, // 1GB
```

Comment:
Isn't it 1073741824 for 1GB? What about writing these kind of constants as 1024 * 1024 * 1024?

Reply:
Bytes is a common convention. Plus if we make it MB, we'll need to convert back to bytes when comparing. 1024*1024*1024 is nice.